### PR TITLE
Add a new Feature request issue template for the project

### PR DESCRIPTION
**Pull request by fortiersteven**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2. app_feature.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "[Feature]: Your short description here"
+labels: ["feature","translation app"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+
+        This template should be uses when you want a new feature available
```
Comment (Mc-muffin) [inline .github/ISSUE_TEMPLATE/2. app_feature.yaml]: Typo "uses" should be "used"

File: .github/ISSUE_TEMPLATE/2. app_feature.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        in the Translation App. Your request will be validated by the lifebottle team and we will
```
Comment (Mc-muffin) [inline .github/ISSUE_TEMPLATE/2. app_feature.yaml]: Maybe Capitalize "LifeBottle"?

**Review by Mc-muffin — COMMENTED** [review]
Just 2 things, otherwise LGTM :)

**Comment by fortiersteven** [comment]
@Mc-muffin I fixed the two typos. Thanks!

**Review by Mc-muffin — APPROVED** [review]

**Pull request merged**